src/js/components/TabPanesComponent.jsx

```diff
@@ -33,15 +47,23 @@ var TabPanesComponent = React.createClass({
 
     return (
       <TogglableTabsComponent activeTabId={this.getTabId()}
-        className="container-fluid">
+                              className="container-fluid">
         <TabPaneComponent id={tabs[0].id}>
-          <Link
+          <div className="row">
```

Comment:
These columns don't collapse nicely on narrow screens - let's remove them. The button is already floated right, and the filter component can be prevented from filling the screen with `max-width: 330px` or similar.